package.json

```diff
@@ -28,6 +28,8 @@
     "lint-staged.root": "lint-staged",
     "lint-staged.workspaces": "npm run lint-staged --workspaces --if-present",
     "new-tokens-data": "npm run snapshot --workspace=packages/genesys-spark-tokens",
+    "playground.angular": "cd ./playground/angular-v13 && rm ./package-lock.json && npm install && npm start",
```

Comment:
I'm going to try using a playground workspace instead of changing directory in this script